sdk/translation/ai-translation-text/samples/v3.0/typescript/src/breakSentence.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Could you please copy those files over to the `samples-dev` folder that should live next to the `samples` folder. Once the sources are there, you can run `npx dev-tool samples publish -f` to regenerate the `samples` folder . Make sure to address the feedback in package.json first. For more information, please refer to [this guide](https://github.com/Azure/azure-sdk-for-js/wiki/Samples-v2-Migration-Guide#migrating-an-existing-package).

Reply:
Awesome, this is done.